src/Configuration/IOptionsExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Configuration
+{
+	/// <summary>
+	/// A collection of extension methods to safely get the configuration values.
+	/// </summary>
+	public static class IOptionsExtensions
+	{
+		/// <summary>
+		/// Returns the <seealso cref="IOptionsMonitor{TOptions}.CurrentValue"/> from the given options.
+		/// <para>
+		/// If the options value does not pass the validation set while configuring the configuration at startup,
+		/// it will return null instead of throwing <seealso cref="OptionsValidationException"/>.
+		/// </para>
+		/// <para><em><strong>Note</strong> - In certain context, handling directly the exception, or let the service fail if the configuration
+		/// is wrong, would be a better solution.</em></para>
+		/// <para><em>Example:</em></para>
+		/// <para>
+		/// <code>
+		///	try
```

Comment:
NIT: Isn't it wrongly formatted?

Reply:
Yes, there was a tab instead of a space, thanks for spotting that